### PR TITLE
LLM - Addition of the Grid Layout to the Flexible Content Cards System

### PR DESCRIPTION
**Pull request by cgrellard-ledger**
### 📝 Description

Addition of the Grid Layout to the Flexible Content Cards System as well as 3 different variants (small, medium and large) of the squared content card

### ❓ Context

- **JIRA or GitHub link**: 

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by sshmaxime — APPROVED** [review]

File: apps/ledger-live-mobile/src/dynamicContent/utils.tsx [hunk 1]
```diff
@@ -141,3 +142,45 @@ export const mapAsHorizontalContentCard = (card: BrazeContentCard): HorizontalCo
...
+});
+
+export const mapAsBigSquareContentCard = (card: BrazeContentCard): VerticalContentCard => ({
+  id: card.id,
+  tag: card.extras.tag,
+  title: card.extras.title,
+  description: card.extras.description,
+  image: card.extras.image,
+  price: card.extras.subtitle,
+  size: "L",
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/dynamicContent/utils.tsx:181]: You can't just give size as parameter ? as only size change ?

**Review by mcayuelas-ledger — COMMENTED** [review]
But nice job 👍

File: apps/ledger-live-mobile/src/contentCards/layouts/grid/index.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        marginHorizontal: "auto",
+        width: "100%",
+        flexDirection: "row",
+        flexWrap: "wrap",
+        gap: styles.gap,
+      }}
+    >
+      {items.map(item => {
+        return (
+          <Flex style={{ width }}>
```
Comment (KVNLS) [inline apps/ledger-live-mobile/src/contentCards/layouts/grid/index.tsx:35]: We need a key here

File: apps/ledger-live-mobile/src/dynamicContent/utils.tsx (hunk 1 above)
Comment (KVNLS) [reply apps/ledger-live-mobile/src/dynamicContent/utils.tsx:181]: We also have a mapping for the price but I agree we could have something reused between the mapping functions

File: apps/ledger-live-mobile/src/dynamicContent/ContentCardsCategory/Layout.tsx [hunk 3]
```diff
@@ -10,10 +10,18 @@ import {
...
+  mapAsBigSquareContentCard,
+} from "~/dynamicContent/utils";
 import Carousel from "../../contentCards/layouts/carousel";
 import useDynamicContent from "../useDynamicContent";
 import { ContentCardsType } from "../types";
+import Grid from "~/contentCards/layouts/grid";
+import VerticalCard from "~/contentCards/cards/vertical";
 
 // TODO : Better type to remove any (maybe use AnyContentCard)
 /* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (KVNLS) [inline apps/ledger-live-mobile/src/dynamicContent/ContentCardsCategory/Layout.tsx:27]: Is this comment still relevant? (As I see that we use AnyContentCard)

**Review by KVNLS — CHANGES_REQUESTED** [review]

File: apps/ledger-live-mobile/src/dynamicContent/ContentCardsCategory/Layout.tsx (hunk 3 above)
Comment (cgrellard-ledger) [reply apps/ledger-live-mobile/src/dynamicContent/ContentCardsCategory/Layout.tsx:27]: Still relevant. We still have an `any` that we need to find a way to remove

**Pull request merged**